imagersite/imager_images/models.py

```diff
@@ -9,16 +10,26 @@
            ('Public', 'Public')]
 
 
+class Face(models.Model):
+    name = models.CharField(max_length=128, blank=True, null=True)
+    x = models.IntegerField()
+    y = models.IntegerField()
+    width = models.IntegerField()
+    height = models.IntegerField()
+
+
 @python_2_unicode_compatible
 class Photo(models.Model):
+    faces = models.ManyToManyField(Face, related_name='photos', blank=True, null=True)
```

Comment:
this probably shouldn't be a many-to-many field.  Any single face box should only be associated with a single photo, no?  Even if the same person appears in more than one photo, they should have a slightly different bounding box.  Change this to be a `ForeignKey` on the `Face` model pointing to `Photo`